File: Array/Interesting Array.go
package main

import "fmt"

func main() {

	arr := []int{9, 17}
	result := checkInterestingArray(arr)
	fmt.Println(result)
}

func checkInterestingArray(arr []int) string {
	count := 0

	for i := 0; i < len(arr); i++ {
		if arr[i]%2 == 1 {
			count++
		}
	}

	if count%2 == 0 {
		return "Yes"
	}

	return "No"
}

File: Pythons Solutions/Interesting Array.py
class Solution:
    def checkSpecialArray(self, arr):
        count = 0

        for i in range(len(arr)):
            if arr[i]%2 == 1:
                count +=1
        
        if count % 2 == 0:
            return "Yes"
        else:
            return "No"

obj = Solution()
arr = [9, 17]
ans = obj.checkSpecialArray(arr)
print(ans)